docs/porting/connectivity/MeshInterface.md

```diff
@@ -176,6 +176,87 @@ State|Description
 
 ![tx](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/tx_process.png)
 
+## MESH PHY extended driver support
```

Comment:
Is this only for people porting Mbed OS?
This file seems to deviate from our porting guide template.

Reply:
Name is MeshInterface? @artokin , @mikter and @JarkkoPaso How we manage this.

Reply:
@JarkkoPaso and @artokin We must add Jarkko's PR commit to this when it is merged our feature branch. Jarkko have re writed porting guide so This comment not need any action now.

Reply:
Commit "RF driver porting guide refactored" added to this PR